man/man8/swtpm.8

```diff
@@ -176,7 +176,8 @@ Use the given port rather than using the environment variable \s-1TPM_PORT.\s0
 .IP "\fB\-t|\-\-terminate\fR" 4
 .IX Item "-t|--terminate"
 Terminate the \s-1TPM\s0 after the client has closed the connection.
-.IP "\fB\-\-server [type=tcp][,port=<port>[,bindaddr=<address>[,ifname=<ifname>]]][,fd=<fd>][,disconnect]\fR" 4
+.IP "\fB\-\-server [type=tcp][,port=<port>[,bindaddr=<address>[,ifname=<ifname>]]]
+\&[,fd=<fd>][,disconnect]\fR" 4
```

Comment:
Did you edit this file directly? Please don't. The next time the man page is recreated the changes will be lost.

Reply:
@stefanberger , thank you for your advice. If then, I would like to change man/man8/swtpm.pod and suggest that add spaces to the long line below. Lintian said the line was too long, so it may have a kind of line break problem.
 
```bash
# This one is original.
=item B<--server [type=tcp][,port=E<lt>portE<gt>[,bindaddr=E<lt>addressE<gt>[,ifname=E<lt>ifnameE<gt>]]][,fd=E<lt>fdE<gt>][,disconnect]>

# This one is that I suggest.
=item B<--server [type=tcp] [,port=E<lt>portE<gt> [,bindaddr=E<lt>addressE<gt> [,ifname=E<lt>ifnameE<gt>] ] ] [,fd=E<lt>fdE<gt>] [,disconnect]>
```
Would you give me some advice about it?

Best regards,

Seunghun

Reply:
Wow, it's great! I will apply this patch and check it again. :)

Reply:
I have checked it with lintian and rebased my patches with your patch. Thank you for helping me, and please review them. Because of regenerating the man/man8/swtpm.8 file, the conflict occurred like below. Would it be alright if I leave it? or should I change the date for solving it?
```bash
<<<<<<< fix/fix_warning_and_typos_in_manpages
.TH swtpm 8 "2020-06-06" "swtpm" ""
=======
.TH swtpm 8 "2020-04-23" "swtpm" ""
>>>>>>> master
```

Reply:
I fixed it.